utils/build/rollup.config.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import path from 'path';
+import commonjs from '@rollup/plugin-commonjs';
+import replace from '@rollup/plugin-replace';
+import nodeResolve from '@rollup/plugin-node-resolve';
+import typescript from 'rollup-plugin-typescript2';
```

Comment:
Was `@rollup/plugin-typescript` considered as an alternative? It seems like it's less popular, but it's an official lib from rollup. `rollup-plugin-typescript2` seems more active though.

Reply:
The official plugin doesn't support the ability to place the type declaration files in a directory provided by the `tsconfig` file (`typings/*` for us). I also found some official examples which used the `rollup-plugin-typescript2` plugin instead 😄

Reply:
Cool, was curious. This looks good!